### ISSUE TITLE
Wrong font color in Manual Search

### ISSUE THREAD

**Issue by eddywitkamp** [pymedusa/Medusa#8743]
**Describe the bug**
Text is hardly readable, white on a light colored background.
See screenshot. 

**To Reproduce**
1. Go to a series episode
2. Click on Manual Search

**Expected behavior**
Black text color

**Screenshots**
![image](https://user-images.githubusercontent.com/6422683/99303082-9f4c3b00-2850-11eb-87f9-591e7537f98c.png)

**Medusa (please complete the following information):**

Medusa Info: | Branch: master
Commit: f39927ea6f0b3d18856dce32d86af43d57713720
Version: 0.4.5
Database: 44.17
Python Version: | 3.7.3 (v3.7.3:ef4ec6ed12, Mar 25 2019, 22:22:05) [MSC v.1916 64 bit (AMD64)]
SSL Version: | OpenSSL 1.1.0j 20 Nov 2018
OS: | Windows-10-10.0.18362-SP0
Locale: | nl_NL.cp1252

**Comment by medariox** [comment]
Force refresh with Ctrl + F5.

**Comment by eddywitkamp** [comment]
That doesn't help

**Comment by medariox** [comment]
Browser?

**Comment by eddywitkamp** [comment]
Google Chrome Versie 86.0.4240.193 (Officiële build) (64-bits)

**Comment by medariox** [comment]
No issue here with the same browser. Try a restart.

**Comment by eddywitkamp** [comment]
Close Chrome (actually there was an update to Chome) - no change

Did a Medusa restart - no change.


Are you sure you are looking at the Manual Search results?

![image](https://user-images.githubusercontent.com/6422683/99315630-36ba8980-2863-11eb-92d9-853f9b7be00b.png)

**Comment by medariox** [comment]
Yes, I'm sure. I will reopen the issue, but I doubt anyone will be able to help you as apparently it can't be reproduced.

**Comment by eddywitkamp** [comment]
Same issue on Firefox.

Medusa won't even load in Edge (old version) :-p

**Comment by p0psicles** [comment]
Maybe best is to backup all db files. main.db, failed.db, cache.db and config.ini.
Then reinstall, and put the files back.

**Comment by ukharley** [comment]
Same for me, unreadable white font. Tried with Chrome and Waterfox.
